assessments/projects/cubefs/self-assessment.md

```diff
@@ -0,0 +1,300 @@
+# CubeFS Self-assessment
+The Self-assessment is the initial document for projects to begin thinking about the
+security of the project, determining gaps in their security, and preparing any security
+documentation for their users. This document is ideal for projects currently in the
+CNCF **sandbox** as well as projects that are looking to receive a joint assessment and
+currently in CNCF **incubation**.
+
+For a detailed guide with step-by-step discussion and examples, check out the free 
+Express Learning course provided by Linux Foundation Training & Certification: 
+[Security Assessments for Open Source Projects](https://training.linuxfoundation.org/express-learning/security-self-assessments-for-open-source-projects-lfel1005/).
+
+# Self-assessment outline
+
+## Table of contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Actors](#actors)
+  * [Actions](#actions)
+  * [Background](#background)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+* [Action Items](#action-items)
+
+## Metadata
+
+|   |  |
+| -- | -- |
+| Software | https://github.com/cubefs/cubefs  |
+| Website | https://cubefs.io/ |
+| Security Provider | No  |
+| Languages | Golang |
+| SBOM | [CubeFS SBOM File](https://github.com/avinashnarasimhan18/tag-security/blob/4e6b73248f7807cef59a456ab87d8f357094f380/assessments/projects/cubefs/cubefs_sbom.json)|

```

Comment:
This link seems to be from your personal fork of the repository.
Please update it with the link from the project.

Reply:
The SBOM Section has been updated.